src/foremast/awslambda/awslambda.py

```diff
@@ -61,6 +61,22 @@ def _check_lambda(self):
             pass
         return exists
 
+    def _check_lambda_alias(self):
+        """ Checks if lambda alias exists
+        Returns:
+            True if alias exists
+            False if alias does not exist
+        """
+        aliases = self.lambda_client.list_aliases(FunctionName=self.app_name)
+
+        for alias in aliases['Aliases']:
+            if alias['Name'] == self.env:
+                LOG.info('Found alias %s for function %s', self.env, self.app_name)
+                return True
```

Comment:
I was just thinking of this for consistency. We were starting to trend more toward one exit point. I'm okay with leaving this though, just thought I would mention.